docs/sitemap.md

```diff
@@ -0,0 +1,34 @@
+# Site map
+
+- Home Page
+  - sizzle
+  - why another design system?
+  - what business problems Mineral solves
+- Getting Started
+  - installation
+  - theming intro
+  - code examples
+- Guidelines
+  - Color (ramps, usage, a11y, interactive elements)
+    - ramps
+    - usage
+    - a11y
+    - interactive color chooser
+  - Typography
+   - usage guidelines (this will evolve as we create the Heading component, for example.)
+  - Tokens / Theming
```

Comment:
Also, is this supposed to be under Guidelines? I think your indentation got messed up either here or at Community / Resources.